packages/firestore/src/core/query.ts

```diff
@@ -349,12 +349,17 @@ export class Query {
     return null;
   }
 
-  hasRelationOpFilter(relationOp: RelationOp): boolean {
-    return (
-      this.filters.find(
-        filter => filter instanceof RelationFilter && filter.op === relationOp
-      ) !== undefined
-    );
+  // Checks if any of the provided RelationOps are included in the query and
+  // returns the first one that is, or null if none are.
+  findRelationOpFilter(relationOps: RelationOp[]): RelationOp | null {
```

Comment:
By the way, I realize this is similar to what you did with getRelationOp() that I asked you to undo...  I _think_ this version is reasonable, since it operates on an array and returns the first match (rather than taking a single op and returning it back to you)...  But I apologize for not suggesting this to you, and sorry if I seem hypocritical / inconsistent. :)